lib/bump/cli/commands/deploy.rb

```diff
@@ -9,12 +9,13 @@ class Deploy < Base
         option :id, default: ENV.fetch("BUMP_ID", ""), desc: "Documentation public id"
         option :token, default: ENV.fetch("BUMP_TOKEN", ""), desc: "Documentation private token"
         option :specification, desc: "Specification of the definition"
+        option :validation, desc: "Validation mode: basic, strict"
```

Comment:
Wondering if it's not safer to pass the allowed values for it and raise early and properly before propagating it. `values: %w(basic strict)` can't help?

Reply:
👍